project_admin/templates/oh_proj_management/home.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'oh_proj_management/base.html' %}
```

Comment:
When moving a templates to a new app, I'd prefer the path for the templates follow the new app's naming convention, for example: `myapp/templates/myapp/foo.html`